gradle/dependencies.gradle

```diff
@@ -72,7 +72,7 @@ def build = [
     errorProneJavac         : "com.google.errorprone:javac:9+181-r4173-1",
     errorProneTestHelpers   : "com.google.errorprone:error_prone_test_helpers:${versions.errorProneApi}",
     checkerDataflow         : "org.checkerframework:dataflow-nullaway:${versions.checkerFramework}",
-    guava                   : "com.google.guava:guava:24.1.1-jre",
+    guava                   : "com.google.guava:guava:30.1-jre",
```

Comment:
This will need some internal testing before release. I do believe we have a later version of Guava in the annotation processor path for at least Java monorepo, but we need to test this for both internal monorepos.